src/components/Calendar.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { LocaleConfig, Calendar as RNCalendar, DateObject } from 'react-native-calendars';
+import { Overlay } from 'react-native-elements';
+
+const i18n = require('../../loc');
+
+LocaleConfig.locales['en'] = {
+  monthNames: [
+    i18n.calendar.monthNames.january,
+    i18n.calendar.monthNames.february,
+    i18n.calendar.monthNames.march,
+    i18n.calendar.monthNames.april,
+    i18n.calendar.monthNames.may,
+    i18n.calendar.monthNames.june,
+    i18n.calendar.monthNames.july,
+    i18n.calendar.monthNames.august,
+    i18n.calendar.monthNames.september,
+    i18n.calendar.monthNames.october,
+    i18n.calendar.monthNames.november,
+    i18n.calendar.monthNames.december,
+  ],
+  monthNamesShort: [
+    i18n.calendar.monthNamesShort.january,
```

Comment:
To short names maybe we could just cut first three letters from `monthNames`

Reply:
We can get those names from `moment.js` I believe

Reply:
or date-fns